docs/input/docs/learn/branching-strategies/gitflow/examples.md

```diff
@@ -6,61 +6,79 @@ RedirectFrom:
 - docs/git-branching-strategies/gitflow-examples
 ---
 
-These examples are using the _default_ configuration with GitVersion. Which is
-[continuous deployment](/docs/reference/modes/continuous-deployment) mode for
-`develop` and [continuous delivery](/docs/reference/modes/continuous-delivery) mode
-for all other branches.
-
-This default configuration allows you to publish CI builds from develop to a CI
-MyGet feed, or another CI feed. Then all other branches are manually released
-then tagged. Read more about this at [version increments](/docs/reference/version-increments).
+These examples are illustrating the usage of the supported `GitFlow` workflow in GitVersion. 
+To enable this workflow, the build-in template [GitFlow/v1](/docs/workflows/GitFlow/v1.json) needs to be referenced in the configuration as following:
+```yaml
+workflow: GitFlow/v1
+mode: ContinuousDelivery
+```
+
+Where 
+the [continuous deployment](/docs/reference/modes/continuous-deployment) mode for no branches, 
+the [continuous delivery](/docs/reference/modes/continuous-delivery) mode for `main`, `support` and `develop` branches and 
+the [manual deployment](/docs/reference/modes/manual-deployment) mode for `release`, `feature`, `hotfix` and `unknown` branches 
+are specified.
+
+This configuration allows you to publish CI (Continuous Integration) builds from `main`, `support` and `develop` branches to an artifact repository. 
+All other branches are manually published. Read more about this at [version increments](/docs/reference/version-increments).
+
+__Notice:__ The _continuous delivery_ mode has been used for the `main` and the `support` branch in this examples (specified as a fallback on the root configuration layer) 
+to illustrate how the version increments are applying. In production context the _continuous deployment_ mode might be a better option when e.g. 
+the releasing process is automated or the commits are tagged by the pipeline automatically.
 
```

Comment:
This section can be wrapped in an alert.

I've also fixed a couple of typos in the text.

```suggestion
:::{.alert .alert-info}
The _continuous delivery_ mode has been used for the `main` and the
`support` branch in this examples (specified as a fallback on the root
configuration layer) to illustrate how the version increments are applied.
In production context the _continuous deployment_ mode might be a better
option when e.g. the release process is automated or the commits are tagged
by the pipeline automatically.
:::
```